common/src/main/java/io/netty/util/NetUtil.java

```diff
@@ -684,46 +648,19 @@ private static boolean isValidIPv4Mapped(byte[] bytes, int currentIndex, int com
      * @return true, if the string represents an IPV4 address in dotted
      *         notation, false otherwise
      */
-    public static boolean isValidIpV4Address(String value) {
+    public static boolean isValidIpV4Address(String ip) {
+        return isValidIpV4Address(ip, 0, ip.length());
+    }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (8) used in the expression (maximum allowed 5). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1067)